PluginsAndFeatures/azure-toolkit-for-intellij/resources/whatsnew/whatsnew.md

```diff
@@ -1,6 +1,27 @@
 <!-- Version: 3.51.0 -->
 # What's new in Azure Toolkit for IntelliJ
 
+## 3.55.0
+### Added
+- New azure resource connector explorer for connection management
+- Add support for IntelliJ proxy
+- Add unified `Open In Portal` support for Web App/VM/Resource Group
```

Comment:
rephrase, the statement 'Add unified `Open In Portal` support' is awkward, and it lacks the place where the `Open In Portal` is added.

Reply:
Snapshot too